cmd/fuseml_core-cli/main.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func main() {
 	var (
-		hostF = flag.String("host", "localhost", "Server host (valid values: localhost)")
+		hostF = flag.String("host", "0.0.0.0", "Server host (valid values: 0.0.0.0)")
```

Comment:
All good, but shouldn't that be in another PR? (it's already in a different commit)

Reply:
For the container to work we need the process to listen in a proper interface. So I just added it to that PR,